enterprise/api/v1/serializers.py

```diff
@@ -1828,3 +1828,111 @@ class EnterpriseGroupLearnersRequestQuerySerializer(serializers.Serializer):
         child=serializers.EmailField(required=True),
         required=False,
     )
+
+
+class EnterpriseUserSerializer(serializers.Serializer):
+    """
+    Serializer for EnterpriseCustomerUser model with additions.
+    """
+
+    class Meta:
+        model = models.EnterpriseCustomerUser
+        fields = (
+            'enterprise_customer_user_id',
+            'user_name',
+            'user_email',
+            'is_admin',
+            'pending_enterprise_customer_user_id',
+            'is_pending_admin'
+        )
+
+    enterprise_customer_user_id = serializers.SerializerMethodField()
+    user_name = serializers.SerializerMethodField()
+    user_email = serializers.SerializerMethodField()
+    is_admin = serializers.SerializerMethodField()
+    pending_enterprise_customer_user_id = serializers.CharField(required=False, default=None)
+    is_pending_admin = serializers.BooleanField(required=False, default=False)
+
+    def get_enterprise_customer_user_id(self, obj):
+        """
+        Get enterprise customer user id
+        """
+        return obj.user_id
+
+    def get_user_name(self, obj):
+        """
+        Get enterprise customer user name
+        """
+        return obj.enterprise_customer.name
+
+    def get_user_email(self, obj):
+        """
+        Get enterprise customer user email
+        """
+        return obj.enterprise_customer.contact_email
```

Comment:
(issue): the user email should be coming from the user and not the enterprise customer.